autoload/ale/handlers/gcc.vim

```diff
@@ -24,7 +24,7 @@ function! ale#handlers#gcc#HandleGCCFormat(buffer, lines) abort
     " <stdin>:8:5: warning: conversion lacks type at end of format [-Wformat=]
     " <stdin>:10:27: error: invalid operands to binary - (have ‘int’ and ‘char *’)
     " -:189:7: note: $/${} is unnecessary on arithmetic variables. [SC2004]
-    let l:pattern = '\v^([a-zA-Z]?:?[^:]+):(\d+):(\d+)?:? ([^:]+): ?(.+)$'
+    let l:pattern = '\v^([a-zA-Z]?:?[^:]+):(\d+):(\d+)?:? ([^:]+): (.+)$'
```

Comment:
Don't forget to undo the changes to this file too.

Reply:
That change was intentional.  I added the question mark to that regular expression when I added the initial Flawfinder support that piggybacked off of the GCC handler in #1361 .   Now that Flawfinder has its own handler, I figured I would revert that back to what it had been before I commandeered the GCC handler for Flawfinder.  I can add it back in if you like, but there shouldn't be anything relying on that question mark anymore.

Reply:
Okay, what about the other changes below?

Reply:
Hmm, not sure how I missed all that.  I'll try to get a fix in later today.